guides/leon/debain-email.md

```diff
@@ -0,0 +1,308 @@
+# Debian Email Box
+
+## Summary 
+This is the setup guide for the Debian Email box. We will be using a Postfix, dovecot, and webmail setup.
+
+### Contact
+- Twitter: @Dubliyu
+- Slack: @yourlocalgod on wcscusf.slack.com
+- Email: [Gmail](mailto:cleonromero@mgail.com)
+
+### Table
+1. [Prerequisites](#pre-reqs)
+2. [Setup Debian VM](#setup-vm)
+3. [Install Postfix](#postfix)
+4. [Install Dovecot](#dovecot)
+5. [Install Webmail](#webmail)
+6. [Testing](#testing)
+
+<a id="pre-reqs"></a>
+## Prerequisites 
+1. Have your [virtual environment](https://silexone.github.io/guides/nestor/ISPsetup.html) configured
+2. Have the [ISP](https://silexone.github.io/guides/nestor/ISPsetup.html) gateway running.
+3. Have pfSense running.
+
+
+<br>
+
+<a id="setup-vm"></a>
+## Setup Debian VM 
+1. First, go get the Debian server from ISO [here](https://cdimage.debian.org/debian-cd/current/amd64/iso-cd/debian-9.3.0-amd64-netinst.iso). 
+    Then, open up VirtualBox, create a new Linux Debian (64-bit) VM, the default setting will do. Then alter the network settings to use Host-only adapter instead of NAT. Insert the downloaded ISO into the virtual optical drive and boot.
+   
+2. Select options as follows in the installation (pretty much a standard setup)
+    1. Choose Graphical install
+    2. English
+    3. United States
+    4. American English
+    5. "debian"
+    6. "debian-email.com"
+    7. {super secret password}
+    8. {some user}
+    9. {some user password}
+    10. Eastern
+    11. Guided - sue entire disk
+    12. All files in one partition
+    13. Finish partitioning
+    14. Yes - write to file
+    15. No
+    16. United States
+    17. ftp.us.debian.org
+    18. No - no popularity contest
+    19. For software selection pick only: standard system utilities
+    20. Yes - to grub
+    21. /dev/sda - should be the only disk
+4. Now you should see a terminal login prompt. Login.
+![login](login.PNG)
+<br>
+
+<a id="postfix"></a>
+## Install Postfix 
+Here we will install Postfix.
+
+1. Preperations
+
+    First become superuser and fetch updates
+    ``` bash
+    su
+    # Enter the {super secret password}
+    apt-get update
+    apt-get upgrade
+    ```
+    
+    Now install some packages that will be useful later
+    ```bash
+    apt-get install -y net-tools wget curl
+    ```
+    
+    Now open up /etc/host.conf file with `vi` configure it so it looks like so.
+    ```bash
+    order hosts, bind
+    multi on
+    ```
+    
+    Now we add our ip and domain name 'debian-email.com' to the host file so it will internally resolve. Make sure someone on the DNS server has debian-email.com point to this box - or add it manually to each windows/linux host file.
+    
+    First, get the IP by running `ifconfig` . The info we want will appear like so  `inet 10.0.2.15` . Now run these two commands, use your IP instead of 10.0.2.15

```

Comment:
This is good, it would be even better if you had more of the output. There's a lot of confusing info for a new user so this will help them replicate what you've done. Also be more verbose with why we do these commands. 

We run `ipconfig` to get our host machines [IP Address](link-to-ipv4-guide) which outputs:

```
eth0      Link encap:Ethernet  HWaddr 00:0B:CD:1C:18:5A
inet addr:172.16.25.126  Bcast:172.16.25.63  Mask:255.255.255.224
inet6 addr: fe80::20b:cdff:fe1c:185a/64 Scope:Link
UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
RX packets:2345583 errors:0 dropped:0 overruns:0 frame:0
TX packets:2221421 errors:0 dropped:0 overruns:0 carrier:0
collisions:0 txqueuelen:1000
RX bytes:293912265 (280.2 MiB)  TX bytes:1044100408 (995.7 MiB)
Interrupt:185 Memory:f7fe0000-f7ff0000
```

As you can see the host machines IP is `172.16.25.126`. This will be need for the next commands.

```bash
 hostnamectl set-hostname mail.debian-email.com
echo "10.0.2.15" debian-email.com mail.debian-email.com >> /etc/hosts
```
This changes the hostname while associating to that IP, which can be seen when you reboot the box.